apps/bluetooth-music/app.js

```diff
@@ -28,6 +30,8 @@ function onAudioFocusGained () {
   if (needResume && !playing) {
     a2dp.play()
   }
+  nowPlayingCenter.setNowPlayingInfo({/** nothing to be set */})
+  nowPlayingCenter.on('command', audioFocus.onRemoteCommand)
```

Comment:
Note that every `nowPlayingCenter.on('command', audioFocus.onRemoteCommand)` would register a new listener to the `EventEmitter`. It should be moved to global scope or the listener would be leaked.